components/LocationCard/index.js

```diff
@@ -0,0 +1,153 @@
+import React, { useEffect, useState } from "react";
+import styled from "styled-components";
+import { useRouter } from "next/router";
+import Link from "next/link";
+import Image from "next/image";
+
+const PageHeader = styled.header`
+  text-align: center;
+  padding: 1rem 0;
+  border-bottom: 1px solid #ccc;
+`;
+
+const HeaderTitle = styled.h1`
+  margin: 0;
+`;
+
+const CardContainer = styled.div`
+  background-color: white;
+  border: 1px solid #ccc;
+  border-radius: 0.5rem;
+  padding: 1rem;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
+  max-width: 500px;
+  margin: 0 auto;
+`;
+
+const CardImage = styled.img`
+  width: 100%;
+  border-radius: 0.5rem;
+  margin-bottom: 1rem;
+`;
+
+const FieldContainer = styled.div`
+  margin-bottom: 1rem;
+  display: flex;
+  align-items: center;
+`;
+
+const Label = styled.label`
+  display: flex;
+  align-items: center;
+  gap: 0.5rem;
+`;
+
+const FieldValue = styled.p`
+  margin: 0;
+`;
+
+const NotesContainer = styled.div`
+  margin-top: 1rem;
+  display: flex;
+  align-items: center;
+`;
+
+const ReturnButtonContainer = styled.div`
+  display: flex;
+  justify-content: center;
+  border-top: 1px solid #ccc;
+  padding-top: 1rem;
+  margin-top: 1rem;
+`;
+
+export default function LocationCard() {
+  const router = useRouter();
+  const [submittedData, setSubmittedData] = useState(null);
+
+  useEffect(() => {
+    const dataFromLocalStorage = JSON.parse(
+      localStorage.getItem("favoriteLocation")
+    );
+    setSubmittedData(dataFromLocalStorage);
+  }, []);
+
+  const handleReturnClick = () => {
+    router.push("/");
+  };
+
+  return (
+    <>
+      <CardContainer>
+        <PageHeader>
+          <HeaderTitle>Location</HeaderTitle>
+        </PageHeader>
+        <CardImage src="/placeholder.svg" alt="Picture" />
```

Comment:
```suggestion
        <CardImage src="/placeholder.svg" alt="Picture" width={100} height={100} />
```
Since it should be a next image component now, a height and width property is required here